docs/config_options.md

```diff
@@ -159,12 +159,12 @@ If you define these options you will enable the associated feature, which may in
   * See [Ignore Mod Tap Interrupt](tap_hold.md#ignore-mod-tap-interrupt) for details
 * `#define IGNORE_MOD_TAP_INTERRUPT_PER_KEY`
   * enables handling for per key `IGNORE_MOD_TAP_INTERRUPT` settings
-* `#define TAPPING_FORCE_HOLD`
-  * makes it possible to use a dual role key as modifier shortly after having been tapped
-  * See [Tapping Force Hold](tap_hold.md#tapping-force-hold)
-  * Breaks any Tap Toggle functionality (`TT` or the One Shot Tap Toggle)
-* `#define TAPPING_FORCE_HOLD_PER_KEY`
-  * enables handling for per key `TAPPING_FORCE_HOLD` settings
+* `#define QUICK_TAP_TERM 100`
+  * double-tap and hold timing to use a dual role key to repeat keycode
```

Comment:
In my opinion, we should strive to differentiate between the action of *tapping* a key and *pressing* a key. Too many people use it interchangeably which complicates the discussions on dual-role key mechanics.

Tapping a key implies pressing it down to send a `key down` keyboard report and then releasing it to send a `key up` keyboard report, while pressing the key is only about pressing the physical key down.

Let 'b' represent `KC_B - key down` and 'B' represent `KC_B - key up`.
A tap is 'bB' and a double tap is 'bBbB'. A press is 'b' and a hold is 'bbbbbbb'.
 In which case "double-tap and hold" would be 'bBbBbbbbbbbbbbbbbbb'.

I think a better description of 'bBbbbbbbbbbb' would be "tap-then-hold" or "double-press hold".

To go on a tangent; what muddies the waters of tap-hold terminology and semantics further is that the "tap" keycode of tap-hold keys doesn't always involve a physical tapping action. The two concepts are so intertwined and yet different enough that when they're used in the same paragraph, you can easily confuse one for the other. Although sadly, I don't see an actionable way to decouple the two in the stage that we're at.

Reply:
I agree with you but unfortunate there isn't a style-guide for describing these actions on documentation so we're left with tap and pressed used interchangeably. That said, "tap-then-hold" sounds more descriptive.